web/main.rb

```diff
@@ -4,31 +4,38 @@
 require 'string-utility'
 require_relative '../utils/utilities'
 
-gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
-demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
-adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
+$gh = Utilities.configure_stuff(token: ENV['GHULS_TOKEN'])
+$demonyms = YAML.load_file("#{Dir.pwd}/web/public/demonyms.yml")
+$adjective_path = "#{Dir.pwd}/web/public/adjectives.txt"
```

Comment:
No problem, @elifoster this should be good to merge in, I tested the code myself with the same change @ProgramFOX made